src/js/stores/UserSettingsStore.ts

```diff
@@ -36,6 +37,27 @@ class UserSettingsStore extends EventEmitter {
   get storeID() {
     return "userSettings";
   }
+
+  get JSONEditorExpandedSetting(): boolean {
+    const settings = (this.getKey(JSON_EDITOR_SETTING) || {}) as {
+      expanded?: boolean;
+    };
+
+    if (settings.expanded && typeof settings.expanded === "boolean") {
```

Comment:
if we have a `typeof` for `boolean` here, we should also guard against `settings` not being an object.

Reply:
done